test/tools/get-doc.test.js

```diff
@@ -183,12 +180,96 @@ describe("get-doc tool", () => {
       },
     });
     assert.equal(isError, undefined);
-    /** @type {string} */
-    const text = content[0].text;
+    const [_, text] = frontmatterSplit(content[0].text);
     assert.ok(text.startsWith("# Headers"));
   });
 
+  describe("bcd keys", () => {
+    it("should have none", async () => {
+      mockPool
+        .intercept({
+          path: "/en-US/docs/Glossary/index.json",
+          method: "GET",
+        })
+        .reply(200, glossaryDoc);
+
+      /** @type {any} */
+      const { content } = await client.callTool({
+        name: "get-doc",
+        arguments: {
+          path: "/en-US/docs/Glossary",
+        },
+      });
+      /** @type {string} */
+      const text = content[0].text;
+      assert.ok(
+        text.startsWith("# Glossary of web terms"),
+        "has no frontmatter",
+      );
+    });
+
+    it("should have one", async () => {
+      mockPool
+        .intercept({
+          path: "/en-US/docs/Web/API/Headers/index.json",
+          method: "GET",
+        })
+        .reply(200, headersDoc);
+
+      /** @type {any} */
+      const { content, isError } = await client.callTool({
+        name: "get-doc",
+        arguments: {
+          path: "/en-US/docs/Web/API/Headers",
+        },
+      });
+      assert.equal(isError, undefined);
+      const [frontmatter] = frontmatterSplit(content[0].text);
+      assert.ok(
+        frontmatter?.split("\n").includes("bcd_key: api.Headers"),
+        "frontmatter includes bcd key",
+      );
+    });
+
+    it("should have multiple", async () => {
+      mockPool
+        .intercept({
+          path: "/en-US/docs/Web/API/Clipboard_API/index.json",
+          method: "GET",
+        })
+        .reply(200, clipboardDoc);
+
+      /** @type {any} */
+      const { content } = await client.callTool({
+        name: "get-doc",
+        arguments: {
+          path: "/en-US/docs/Web/API/Clipboard_API",
+        },
+      });
+      const [frontmatter] = frontmatterSplit(content[0].text);
+      const lines = frontmatter?.split("\n");
+      assert.partialDeepStrictEqual(lines, [
+        "bcd_keys:",
+        "  - api.Clipboard",
+        "  - api.ClipboardEvent",
+        "  - api.ClipboardItem",
+      ]);
+    });
+  });
+
   after(() => {
     server.listener.close();
   });
 });
+
+/**
+ * @param {string} text
+ * @returns {[string?, string]}
+ */
+function frontmatterSplit(text) {
+  const delim = "---\n";
```

Comment:
Nit: Doesn't the delimiter have newlines on both sides?
```suggestion
  const delim = "\n---\n";
```